Cargo.toml

```diff
@@ -37,13 +34,13 @@ rand = "0.7.3"
 # Feature detection won't work here
 [target.'cfg(any(windows, unix))'.dependencies]
 libc = "0.2.47"
-memmap = { package ="mapr", version = "0.8.0" }
+memmap = { package ="memmap2", version = "0.5" }
```

Comment:
With aot removed, I don't think we would be needing those 2 crates anymore, correct?

Reply:
That's right

Reply:
I was actually referring to **2** crates here, we can also safely remove libc. There's one reference of `c_uchar` in the code but std [has](https://doc.rust-lang.org/std/os/raw/type.c_uchar.html) it already.

Reply:
Done, libc has also been removed